projects/storefrontlib/cms-components/product/product-list/product-list-item/product-list-item.component.html

```diff
@@ -50,7 +50,10 @@ <h2 [innerHtml]="product.nameHtml"></h2>
       </div>
       <div class="col-12 col-md-4">
         <ng-template [cxOutlet]="ProductListOutlets.ITEM_ACTIONS">
-          <ng-container cxInnerComponentsHost></ng-container>
+          <ng-container
+            cxInnerComponentsHost
+            [cxInnerComponentsContext]="{ product: product }"
```

Comment:
use shortcut? `{ product}`

Reply:
or just `product`, not wrappered.

Reply:
In general, for being future-proof and extendable, I'm all for wrapping contexts and any payloads in general in the wrapper objects. Why: we never know if in the future we (or customer) want to pass more data entities into the payload